vowpalwabbit/vw.sln

```diff
@@ -207,6 +209,17 @@ Global
 		{E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|Win32.ActiveCfg = Release|x64
 		{E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|x64.ActiveCfg = Release|x64
 		{E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|x64.Build.0 = Release|x64
+		{DFD56302-EC7E-4480-9A44-A0467887ED20}.Debug|Mixed Platforms.ActiveCfg = Debug|Win32

```

Comment:
It just sets x64 as the build target.  I thought I excluded my vw.sln from any commits.  Maybe I failed to include it when I reverted the dmTest project.  Sorry… will try again.

D

From: John [mailto:notifications@github.com]
Sent: Saturday, November 7, 2015 11:39 AM
To: JohnLangford/vowpal_wabbit <vowpal_wabbit@noreply.github.com>
Cc: Dan Melamed <melamed@microsoft.com>
Subject: Re: [vowpal_wabbit] fix buggy exp(-inf) on Windows (#848)


In vowpalwabbit/vw.sln<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f848%23discussion_r44214177&data=01%7c01%7cmelamed%40microsoft.com%7c6dccba31efe74f3b424108d2e791ed8a%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=euQu4aZQFIxWLTp1mdRzCC3f2PQXsgpOut0OuwcA28A%3d>:

> @@ -207,6 +209,17 @@ Global

>              {E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|Win32.ActiveCfg = Release|x64

>              {E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|x64.ActiveCfg = Release|x64

>              {E4E962AE-7056-4EB0-A8C5-8DC824A4B068}.Release|x64.Build.0 = Release|x64

> +            {DFD56302-EC7E-4480-9A44-A0467887ED20}.Debug|Mixed Platforms.ActiveCfg = Debug|Win32

What does all of this do?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fJohnLangford%2fvowpal_wabbit%2fpull%2f848%2ffiles%23r44214177&data=01%7c01%7cmelamed%40microsoft.com%7c6dccba31efe74f3b424108d2e791ed8a%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=cg%2bxq%2fLCiCYtiMB0%2bkCSoA2drpNUf9T0CHtGFibe%2fOk%3d>.